integration_test/Bindings/Python/dialects/om.py

```diff
@@ -82,6 +85,8 @@
 print(obj.child.foo)
 # CHECK: ('Root', 'x')
 print(obj.reference)
+# CHECK: 14
```

Comment:
Can we add a test to print the whole tuple? Unless it prints as a pointer.

Reply:
Unfortunately it prints a pointer. I'll implement __str__ if necessary in the future